src/components/course/data/utils.jsx

```diff
@@ -817,10 +835,16 @@ export function getEntitlementPrice(entitlements) {
  * @returns Price for the course run.
  */
 export function getCoursePrice(course) {
+  if (course.activeCourseRun?.fixedPriceUsd) {
```

Comment:
nit: exists in the existing conditionals too (e.g., `if (course.activeCourseRun?.fixedPriceUsd)`, but if this is `0` the conditional wouldn't be truthy, and would fallback to `firstEnrollablePaidSeatPrice`. Same conditional fallback with `0` could apply there too, ultimately falling back to `null` despite having a legit `0` price.